packages/cozy-client/src/models/account.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 import merge from 'lodash/merge'
+import CozyClient from '../CozyClient'
```

Comment:
If this is just for the JSDoc, don't import it like that. https://github.com/cozy/cozy-client/pull/1320#discussion_r1113034764